Formula/cln.rb

```diff
@@ -35,6 +35,10 @@ class Cln < Formula
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "wget" => :build
+
+    on_system :linux, :macos_ventura_or_newer do
```

Comment:
```suggestion
    on_system :linux, macos: :ventura_or_newer do
```

Reply:
Sorry for that.
I was checking the guidelines before and I have pushed all commits with the flag `-f`.
Finally, these modifications are accepted.